package.json

```diff
@@ -57,23 +57,23 @@
 		"version_format": "<electron major version>.<package major>.<arbitrary patch>",
 		"nativeModules": {
 			"@parcel/watcher": {
-				"version": "31.2.23",
+				"version": "30.2.1",
```

Comment:
develop has already 2.3.0 for @parcel/watcher and this patch bump will only impact the repackaged module under @hackolade org specifically for Electron 30.  Electron 31 is ready already.

Reply:
All other changes are there to contextualize develop back on Electron 30